zerver/openapi/zulip.yaml

```diff
@@ -8261,6 +8266,71 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /user_topics:
+    post:
+      operationId: update-user-topic
+      summary: Update personal preferences for a topic
+      tags: ["streams"]
+      description: |
+        This endpoint is used to update the personal preferences for a topic,
+        such as the topic's visibility policy, which is used to implement
+        [mute a topic](/help/mute-a-topic) and related features.
+
+        This endpoint can be used to update the visibility policy for the single
+        stream and topic pair indicated by the parameters for a user.
+
+        **Changes**: New in Zulip 7.0 (feature level 170).
+        The `/users/me/subscriptions/muted_topics` endpoint, which used to only
+        mute or unmute a topic, is now deprecated.
+        It exists for backwards compatibility and will be removed once clients
+        have migrated to use this endpoint.
```

Comment:
I propose replacing everything after `170).` with the following, to avoid "unmute a topic" as a phrase, just to avoid confusion with the new feature that endpoint didn't support.

"Previously, toggling the muting state for a topic was managed by the
`/users/me/subscriptions/muted_topics` endpoint, which this endpoint
is intended to replace."

(The other motivation here is to avoid having text here that we'll need to remember to update when we do remove the old endpoint).

Reply:
Updated